pkg/router/template/plugin.go

```diff
@@ -197,14 +202,28 @@ func (p *TemplatePlugin) Commit() error {
 
 // endpointsKey returns the internal router key to use for the given Endpoints.
 func endpointsKey(endpoints *kapi.Endpoints) string {
-	return fmt.Sprintf("%s/%s", endpoints.Namespace, endpoints.Name)
+	return fmt.Sprintf("%s%s%s", endpoints.Namespace, endpointsKeySeparator, endpoints.Name)
```

Comment:
Should this just call endpointsKeyFromParts?

Reply:
Yes, I added endpointsKeyFromParts later out of necessity and didn't release it can be reused here.